types/component/AnonymousChat/BaseChannelItem.types.ts

```diff
@@ -1,10 +1,27 @@
 /* eslint-disable no-shadow */
 export enum BaseChannelItemTypeProps {
+  SIGNED_PM = 'SIGNED_PM',
   ANON_PM = 'ANON_PM',
-  ANON_POST_NOTIFICATION = 'ANON_POST_NOTIFICATION',
+
+  MY_SIGNED_POST_NOTIFICATION = 'MY_SIGNED_POST_NOTIFICATION',
+  MY_SIGNED_POST_NOTIFICATION_I_COMMENTED = 'MY_SIGNED_POST_NOTIFICATION_I_COMMENTED',
+  MY_SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY = 'MY_SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY',
+  MY_SIGNED_POST_NOTIFICATION_COMMENTED = 'MY_SIGNED_POST_NOTIFICATION_COMMENTED',
+  MY_SIGNED_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY = 'MY_SIGNED_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY',
+
   MY_ANON_POST_NOTIFICATION = 'MY_ANON_POST_NOTIFICATION',
   MY_ANON_POST_NOTIFICATION_I_COMMENTED = 'MY_ANON_POST_NOTIFICATION_I_COMMENTED',
-  ANON_POST_NOTIFICATION_I_COMMENTED = 'ANON_POST_NOTIFICATION_I_COMMENTED'
+  MY_ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY = 'MY_ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY',
+  MY_ANON_POST_NOTIFICATION_COMMENTED = 'MY_ANON_POST_NOTIFICATION_COMMENTED',
+  MY_ANON_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY = 'MY_ANON_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY',
+
+  SIGNED_POST_NOTIFICATION = 'SIGNED_POST_NOTIFICATION',
+  SIGNED_POST_NOTIFICATION_I_COMMENTED = 'SIGNED_POST_NOTIFICATION_I_COMMENTED',
+  SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY = 'SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY',
+
+  ANON_POST_NOTIFICATION = 'ANON_POST_NOTIFICATION',
+  ANON_POST_NOTIFICATION_I_COMMENTED = 'ANON_POST_NOTIFICATION_I_COMMENTED',
+  ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY = 'ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new enum values added to `BaseChannelItemTypeProps` seem to be well defined and follow the existing naming convention. However, it's important to ensure that these new types are handled correctly wherever this enum is used in your codebase.
```diff
+  SIGNED_PM = 'SIGNED_PM',
+  MY_SIGNED_POST_NOTIFICATION = 'MY_SIGNED_POST_NOTIFICATION',
+  MY_SIGNED_POST_NOTIFICATION_I_COMMENTED = 'MY_SIGNED_POST_NOTIFICATION_I_COMMENTED',
+  MY_SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY = 'MY_SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY',
+  MY_SIGNED_POST_NOTIFICATION_COMMENTED = 'MY_SIGNED_POST_NOTIFICATION_COMMENTED',
+  MY_SIGNED_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY = 'MY_SIGNED_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY',
+  MY_ANON_POST_NOTIFICATION = 'MY_ANON_POST_NOTIFICATION',
+  MY_ANON_POST_NOTIFICATION_I_COMMENTED = 'MY_ANON_POST_NOTIFICATION_I_COMMENTED',
+  MY_ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY = 'MY_ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY',
+  MY_ANON_POST_NOTIFICATION_COMMENTED = 'MY_ANON_POST_NOTIFICATION_COMMENTED',
+  MY_ANON_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY = 'MY_ANON_POST_NOTIFICATION_COMMENTED_ANONYMOUSLY',
+  SIGNED_POST_NOTIFICATION = 'SIGNED_POST_NOTIFICATION',
+  SIGNED_POST_NOTIFICATION_I_COMMENTED = 'SIGNED_POST_NOTIFICATION_I_COMMENTED',
+  SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY = 'SIGNED_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY',
+  ANON_POST_NOTIFICATION = 'ANON_POST_NOTIFICATION',
+  ANON_POST_NOTIFICATION_I_COMMENTED = 'ANON_POST_NOTIFICATION_I_COMMENTED',
+  ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY = 'ANON_POST_NOTIFICATION_I_COMMENTED_ANONYMOUSLY'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->